kola/tests/kubernetes/kubelet_wrapper.go

```diff
@@ -49,6 +49,8 @@ systemd:
       # that rkt runs the kubelet successfully, we haven't hit this regression,
       # so just printing the version is enough.
       ExecStart=/usr/lib/flatcar/kubelet-wrapper --version
+      Restart=always
```

Comment:
For `oneshot` we cannot have `Restart=always`

Reply:
Oh, I always forget that and how to do it best, maybe change the service type?

Reply:
Updated it

Reply:
We can change the service type, or change to `Restart=on-failure`?